src/Promitor.Core.Scraping/Configuration/Serialization/Deserializer.cs

```diff
@@ -45,7 +45,8 @@ public virtual TObject Deserialize(YamlMappingNode node, IErrorReporter errorRep
                 }
                 else
                 {
-                    errorReporter.ReportWarning(child.Key, $"Unknown field '{child.Key}'.");
+                    errorReporter.ReportWarning(
```

Comment:
Nit suggestion - Maybe not call a method inline?

Reply:
Sure - I've pulled the message out to a local variable. Hope that's what you meant.